x-pack/qa/smoke-test-plugins/build.gradle

```diff
@@ -21,6 +21,10 @@ testClusters.integTest {
       // Do not attempt to install ingest-attachment in FIPS 140 as it is not supported (it depends on non-FIPS BouncyCastle)
       return
     }
+    if (pluginName == 'quota-aware-fs') {
```

Comment:
Same comment as before. Anywhere we are iterating over plugins like this it should be for isolated plugins? All bootstrap plugins will require packaging tests.

Reply:
I talked with Rene about this, but it turned out not to be trivial. Maybe we can revisit this if/when we add another bootstrap plugin?